reana_client/cli/secrets.py

```diff
@@ -63,49 +65,22 @@ def secrets_add(from_literal, from_file, overwrite, access_token):  # noqa: D301
 
     secrets = {}
```

Comment:
Let's call this in a different way, such as `secrets_` to avoid shadowing `secrets` library.